.github/workflows/build_and_test_workflow.yml

```diff
@@ -15,6 +15,8 @@ on:
     paths-ignore:
       - '**/*.md'
       - 'docs/**'
+      - '.lycheeignore'
+      - 'changelogs/fragments/**'
 
```

Comment:
Hi @sejli, don't we have to add the paths ignore for Push/merge trigger as well?